extensions/blocks/recurring-payments/index.js

```diff
@@ -83,3 +88,71 @@ export const SUPPORTED_CURRENCY_LIST = [
 	'SEK',
 	'SGD',
 ];
+
+/**
+ * Stripe's minimum transaction amounts per currency in the smallest unit for
+ * that currency.
+ *
+ * https://stripe.com/docs/currencies#minimum-and-maximum-charge-amounts
+ *
+ * @type { [currenty: string]: number }
+ */
+export const MINIMUM_TRANSACTION_PER_CURRENCY = {
```

Comment:
Can we combine this with `SUPPORTED_CURRENCY_LIST` ?
Something like 
```
export const SUPPORTED_CURRENCY_LIST = [
	'USD': { 'minimumCharge': 0.5 },
```

I am afraid of a situation where somebody adds a new currency without introducing minimum charge

Reply:
There is now a single list. One thing to note, this change now adds more supported currencies: `INR`, `MYR`, and `PLN`. I don't know if that's desired.